.circleci/config.yml

```diff
@@ -333,61 +333,62 @@ workflows:
           filters:
             branches:
               only:
-                - main
+                - ew/plugin-info-hooks-DISABLED
       - release-management/release-package:
-          tag: latest-rc
-          github-release: true
+          tag: dev
+          prerelease: true
+          github-release: false
```

Comment:
Set this to false assuming it would create a release at https://github.com/salesforcecli/sfdx-cli/releases